java/tools/src/test/org/apache/orc/tools/check/TestCheckTool.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.orc.tools.bloomfilter;
+package org.apache.orc.tools.check;
```

Comment:
May I ask why do we need this new test package when the original main code is in `package org.apache.orc.tools`?

Reply:
BTW, this design is not this PR's issue. I checked the existing code and found that we need to revise the existing test code package too.

Reply:
> why do we need this new test package when the original main code is in `package org.apache.orc.tools`?

I'm sorry about that.

I tried to recall why it was designed like this at the beginning, but I kind of forgot.